### ISSUE TITLE
Is mixing Transparent Window Option with Border Window Option possible?

### ISSUE THREAD

**Issue by Kodaigan** [skotlex/ffxiv-material-ui#230]
Hello skotlex,

long-time user of your excellent mod. Been using it ever since I have found it, and am coming back with a new question.

As I was updating the files/mods I was wondering if it was possible to install the transparent window option together with the Border Window option? The result would be that unfocused window would stay transparent, and the focused window would become bordered. (Essentially mixing two options into one.)

Is this possible? My guess is I would have to install the mods in a specific sequence to get it to work. But wanted to ask before I start experimenting.

(Unless mixing was indeed intended from the start.)

Many Thanks
Koda

**Comment by skotlex** [comment]
It is possible if you use `Selected window border + Nonselected window transparency` combination.

* Selected window border: WindowA_BgSelected* (Corner/H/HV/V)
* Nonselected window transparency: WindowA_BgNormal* (Corner/H/HV/V)

**Comment by Kodaigan** [comment]
I think I made it work. But I can't test it right now, as the launcher stopped working today. It keeps giving me the error "504 - https system error". 

I'll update this ticket once I managed to test if it works or not. 
I also attached the file as a zip (github doesn't like unknown file types) if anyone interested would like to test if it works or not.
[discord-selected&transparent-window.zip](https://github.com/skotlex/ffxiv-material-ui/files/6845088/discord-selected.transparent-window.zip)
